databricks_cli/sdk/service.py

```diff
@@ -232,15 +322,34 @@ def __init__(self, client):
 
     def list_clusters(self, headers=None):
         _data = {}
-
         return self.client.perform_query('GET', '/clusters/list', data=_data, headers=headers)
 
-    def create_cluster(self, num_workers=None, autoscale=None, cluster_name=None, spark_version=None,
-                       spark_conf=None, aws_attributes=None, node_type_id=None,
-                       driver_node_type_id=None, ssh_public_keys=None, custom_tags=None,
-                       cluster_log_conf=None, spark_env_vars=None, autotermination_minutes=None,
-                       enable_elastic_disk=None, cluster_source=None, instance_pool_id=None,
-                       headers=None):
+    def create_cluster(
+        self,
+        num_workers=None,
+        autoscale=None,
+        cluster_name=None,
+        spark_version=None,
+        spark_conf=None,
+        aws_attributes=None,
+        azure_attributes=None,
+        gcp_attributes=None,
```

Comment:
What about adding them after headers? Because it's again change in the signature

Reply:
That would mean we can never add fields to the existing functions. Only way to circumvent is by using some kind of builder, but that's too big of a departure from what we're working with today.

Reply:
no, it's ok to add them - just at the end, so it won't break existing things if people are using positional parameters